app/models/manageiq/providers/openstack/cloud_manager/template.rb

```diff
@@ -76,7 +76,9 @@ def requires_storage_for_scan?
 
   def self.raw_create_image(ext_management_system, create_options)
     ext_management_system.with_provider_connection(:service => 'Image') do |service|
-      service.create_image(create_options)
+      url = create_options.delete(:url)
+      image = service.images.create(create_options)
+      service.handle_upload(image, url)
```

Comment:
hm, what was issue with this approach? This was letting the OpenStack to fetch the url, so MIQ doesn't need to fetch all of the data (the images can be quite large easily 50GB and more)

Reply:
Letting OpenStack to fetch url is supported only in Image v1, and deprecated in Image v2

Reply:
As Alexander wrote, passing URL doesn't work e.g. on OSP13.